packages/@expo/metro-config/src/customizeFrame.ts

```diff
@@ -33,7 +33,7 @@ export const INTERNAL_CALLSITES_REGEX = new RegExp(
     // Babel helpers that implement language features
     'node_modules/@babel/runtime/.+\\.js$',
     // Hide Hermes internal bytecode
-    '/(?:InternalBytecode/)?InternalBytecode\\.js$',
+    '/(?:InternalBytecode\/)?InternalBytecode\\.js$',
```

Comment:
```suggestion
    '(?:InternalBytecode/)?InternalBytecode\\.js$',
```

It isn't the `/` here that's a problem, since the escape is redundant for programmatic regex instantiation. Instead, the leading slash is most likely responsible for the mismatch based on the error given, if I had to guess.